usaspending_api/search/tests/integration/test_spending_by_award.py

```diff
@@ -1148,6 +1150,71 @@ def _test_correct_response_for_recipient_id(client):
     assert resp.json().get("results") == expected_result, "Recipient ID filter does not match expected result"
 
 
+def _test_correct_response_for_def_codes(client):
```

Comment:
Since the test fixtures are pretty cluttered for these tests, can we add some extra tests that search for a different DEFC to ensure the award is NOT returned?